src/pages/questions.js

```diff
@@ -20,18 +23,25 @@ export default function QuestionPage() {
 					content="Here you can see all the Questions"
 				/>
 			</Helmet>
+
 			<Button
 				greyButton={true}
 				onClick={() => {
 					setFilter(!filter);
 				}}
 			>
-				{!filter ? `Show Closed` : `Show Open`}
+				{filter ? `Show All Questions` : `Show My Questions`}
 			</Button>
 			<article>
 				{questions.map(question => {
-					if (question.answered === filter) {
-						return <QuestionComponent key={question.id} question={question} />;
+					if (filter ? question.userID === activeUser.userID : true) {
+						return (
+							<QuestionComponent
+								key={question.id}
+								ownedQuestion={question.userID === activeUser.userID ? true : false}
```

Comment:
won't it work just the same without the `if` statement if you added the following here:
```suggestion
								ownedQuestion={filter && (question.userID === activeUser.userID)}
```
Only if both `filter` and `question.userID === activeUser.userID` are true, will `true` be passed.